src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/model/JJDispute.java

```diff
@@ -1,13 +1,25 @@
 package ca.bc.gov.open.jag.tco.oracledataapi.model;
```

Comment:
We also need the fine Reduction reason, the time to pay reason and the offence location as part of this model.  The reconciliation process should pull in the written reasons from our TCO database.  The offence location is on the written ticket as the 'At or near highway', 'at or near city/town' .  We currently do not get our OCR reader to scan this on the handwritten tickets though there is a spot for it.  We may need to follow up about this one in standup.

Reply:
I also added the C# changes to the staff-api and the client side changes to make this PR more complete.  Let me know when you have added the additional model fields for reasons and lets discuss the offence location with the business.